CDCgov/prime-reportstream
# Bug: fix prime bash script for new gradle location

Updates the prime bash script

Test Steps:
1. Run `./gradlew :prime-router:clean`
2. Run `./prime login` just to make sure it builds the jar

## Changes
- Fixes the pathing for building the prime cli

## Checklist

### Testing

### Process

## Linked Issues


## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
